modules/ROOT/pages/deploy-resource-allocation.adoc

```diff
@@ -68,17 +68,24 @@ Performance test results are based on an Anypoint Runtime Fabric cluster with co
 
 A C++ based load generator with better efficiency in SSL connections was used to yield the maximum throughput.
```

Comment:
Just to avoid the hyphenation issue...:
```suggestion
A load generator based on C++, which has better efficiency in SSL connections, was used to yield the maximum throughput.
```